x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferencePlugin.java

```diff
@@ -179,6 +180,12 @@ public class InferencePlugin extends Plugin
         Setting.Property.NodeScope,
         Setting.Property.Dynamic
     );
+    public static final Setting<TimeValue> INFERENCE_QUERY_TIMEOUT = Setting.timeSetting(
+        "xpack.inference.query_timeout",
+        TimeValue.timeValueSeconds(TimeUnit.SECONDS.toSeconds(10)),
```

Comment:
We've typically used 30 seconds as the default. Is that too long, is that why we're using 10 here for semantic text?